source/deployment/sso-saml.rst

```diff
@@ -48,21 +40,21 @@ installed. If not, run - ``apt-get install libxmlsec1-openssl`` on
 Ubuntu - ``yum install xmlsec1-openssl`` on RHEL
 
```

Comment:
If someone deploys SAML with another IDP like OneLogin, they wouldn't likely read through the Okta and ADFS guides.

Hence, for those admins, it is useful to include the pre-installation note here too, as otherwise it might be missed. Similarly with the troubleshooting section,